### ISSUE TITLE
[demo] make translating easier

### ISSUE THREAD

**Issue by Haroenv** [bullgit/blah-code#9]
I'd like it if regardless of code/decode one field would be for regular text and the other one for encoded text, that way you could even get rid of the `encode/decode` button altogether and just perform the action of the last `input` in a field.

**Comment by mischah** [comment]
Yeah. that could make sense. would love to try that in case you would like to create a PR.
